src/libs/CardUtils.ts

```diff
@@ -194,6 +195,16 @@ function getCardFeedIcon(cardFeed: string): IconAsset {
     return Illustrations.AmexCompanyCards;
 }
 
+function getDescriptionForPolicyDomainCard(domainName: string) {
+    const match = domainName.match(CONST.REGEX.EXPENSIFY_POLICY_DOMAIN_NAME);
+    if (match) {
```

Comment:
@mountiny    Should we use `bank` field to check? If possible, I think it will be simpler
<img width="443" alt="Screenshot 2024-09-10 at 00 13 09" src="https://github.com/user-attachments/assets/4b0f01aa-389b-40ca-82b7-6fdadb22b423">

Reply:
We already implemented `isExpensifyCard` function to check this

Reply:
I think we need to change this, it does not have to be only Expensify cards here, I think we need to check if this feed is a workspace feed or not. 

If yes, use workspace name

Reply:
Oh wait, I think @mountiny agrees with your current solution